.gitignore

```diff
@@ -1,6 +1,13 @@
 *.log
 *.tsbuildinfo
-node_modules
+*.vsix
 .DS_Store
+.eslintcache
 .loom
+.next
 .shadowenv.d
+build
+build-internal
```

Comment:
Maybe we should put these in a `.gitignore` in `polaris-react`

Reply:
`*.scss.d.ts` could be added globally and removed from `polaris-react` `.gitignore`.

Reply:
I removed all other `.gitignore` files in favor of having a single ignore file

Reply:
> `*.scss.d.ts` could be added globally

Do we need to add it if it is already part of the build output? Just trying to keep the ignores trim